interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/patient-portal.twig

```diff
@@ -0,0 +1,4 @@
+<script src="{{ assetPath }}../index-portal.php?action=get_telehealth_settings"></script>
+<link rel="stylesheet" href="{{ assetPath }}css/telehealth.css" type="text/css" />
+<script src="{{ assetPath }}js/telehealth.js"></script>
+<script src="{{ assetPath }}js/telehealth-patient.js"></script>
```

Comment:
could do cache busting by makeing a core twig function akin to the smarty custom assetVersionNumber function

Reply:
Added in cache busting.